Normalize CredentialsContainer.create() method (#33360)
mdn/content: files/en-us/web/api/passwordcredentialinit/index.md

---
title: PasswordCredentialInit
slug: Web/API/PasswordCredentialInit
page-type: web-api-interface
spec-urls: https://w3c.github.io/webappsec-credential-management/#typedefdef-passwordcredentialinit
---

> {{APIRef("Credential Management API")}}{{SecureContext_Header}}

Reviewer: I'm not sure about `{{SecureContext_Header}}`. You're supplying this thing and can create it anywhere. On the other hand, you can't use this without `create()` which requires the secure context.

I don't know the "rule" nowdays, but I'd omit it.

Author: Yeah, agreed.

Author: -> https://github.com/mdn/content/pull/33360/commits/bef0047e1197464c69b8e999591c2bdd75a0f9eb

> The **`PasswordCredentialInit`** dictionary represents the object passed to {{domxref("CredentialsContainer.create()")}} as the value of the `password` option: that is, when using `create()` to create a password credential.

Reviewer: Perhaps

```suggestion
The **`PasswordCredentialInit`** dictionary represents the object passed to {{domxref("CredentialsContainer.create()")}} as the value of the `password` option, when creating a password credential.
```

Author: -> https://github.com/mdn/content/pull/33360/commits/ea9b0efc2887f5feeca14a9ca5e028b37c3f3924
